config.py

```diff
@@ -159,19 +159,31 @@ class Config:
             "offline_warning_enabled": True,
             "publish_benchmark_results": True,
             "max_builds": 1,
-            "build_timeout": 150,
+            "build_timeout": 120,
         },
         "ec2-t3-xlarge-us-east-2": {
-            "info": "Supported benchmark langs: Python, R. Runs only benchmarks with cloud = True",
+            "info": "Supported benchmark langs: Python, R, JavaScript",
             "default_filters": {
                 "arrow-commit": {
                     "langs": {
                         "Python": {
                             "names": [
+                                "dataset-filter",
                                 "dataset-read",
                                 "dataset-select",
+                                "file-read",
+                                "recursive-get-file-info",
+                                "wide-dataframe",
+                            ]
+                        },
+                        "R": {
+                            "names": [
+                                "dataframe-to-table",
+                                "file-read",
+                                "partitioned-dataset-filter",
                             ]
                         },
+                        "JavaScript": {"names": ["js-micro"]},
```

Comment:
Have to wait until https://github.com/apache/arrow/pull/41031 is merged to merge this

Reply:
actually I'll just comment this out and merge so we can start collecting data